### PR TITLE
Fix Issue 23261 - druntime core.std.attribute.Tagged1_2 constructor i…

### PR DESCRIPTION
**Pull request by RazvanN7**
…s unsafe

### CODE REVIEW

**Review by thewilsonator — APPROVED** [review]

File: druntime/src/core/attribute.d [hunk 1]
```diff
@@ -236,9 +236,9 @@ version (UdaGNUAbiTag) struct gnuAbiTag
 {
     string[] tags;
 
-    this(string[] tags...)
+    this(string[] tags...) @safe
```
Comment (PetarKirov) [inline druntime/src/core/attribute.d:239]:
```suggestion
    this(string[] tags...) @safe pure nothrow
```

**Comment by RazvanN7** [comment]
@PetarKirov done.

**Comment by Geod24** [comment]
As mentioned in the issue, this code is only intended to be used as an UDA, so this issue doesn't make sense IMO.

**Comment by PetarKirov** [comment]
@Geod24 it's true that whether a problem exists at runtime or only theoretically at compile-time drastically changes the stakes. That said, I see no reason to keep an obviously unsafe code (even if the unsafety is theoretical) given how easy the fix is.

**Pull request merged**